src/modules/renderGraphiQL.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Mostly taken straight from express-graphql, so see their licence
+ * (https://github.com/graphql/express-graphql/blob/master/LICENSE)
+ */
+
+// TODO: in the future, build the GraphiQL app on the server, so it does not
+// depend on any CDN and can be run offline.
+
+export type GraphiQLData = {
```

Comment:
@HriBB I want to take one more day to review the code and discuss the API, but after that, I think we should be ready to go. If you have time to read the code and leave comments, that would be great!